packages/history-utility/src/helpers.ts

```diff
@@ -0,0 +1,7 @@
+const isProduction = import.meta?.env?.MODE === 'production';
```

Comment:
This doesn't work well for minification. We should simply have inline `if (...)`.

Reply:
updated